[doc] Provisioning Admission Check Controller (#1242)
kubernetes-sigs/kueue: site/content/en/docs/admission-check-controllers/_index.md

---
title: "Admission Checks"
linkTitle: "Admisssion Checks"
weight: 10
description: >

>   This section describes the Admission Checks that are currently patr of Kueue.

Reviewer: ```suggestion
  This section describes the Admission Checks that are included with Kueue.
```

---

> Check the [Concepts](/docs/concepts) section for the generic [Admission Check](/docs/concepts/admission_check/) description.

Reviewer: ```suggestion
You can read the [Concepts](/docs/concepts) section to learn how [Admission Checks](/docs/concepts/admission_check/) influence admission.
```
